seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -0,0 +1,74 @@
+import mock
+import responses
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from seqr.views.utils.test_utils import TEST_TERRA_API_ROOT_URL
+from seqr.views.utils.terra_api_utils import list_anvil_workspaces, sa_get_workspace_acl, TerraAPIException
+from seqr.views.utils.test_utils import GOOGLE_API_TOKEN_URL, GOOGLE_SERVICE_ACCOUNT_INFO, GOOGLE_TOKEN_RESULT, GOOGLE_ACCESS_TOKEN_URL
+
+
+@mock.patch('seqr.views.utils.terra_api_utils.TERRA_API_ROOT_URL', TEST_TERRA_API_ROOT_URL)
+@mock.patch('seqr.views.utils.terra_api_utils.GOOGLE_SERVICE_ACCOUNT_INFO', GOOGLE_SERVICE_ACCOUNT_INFO)
+@mock.patch('seqr.views.utils.terra_api_utils.time')
+class TerraApiUtilsCase(TestCase):
+    fixtures = ['users', 'social_auth']
+
+    def setUp(self):
+        self.user = User.objects.get(email = 'test_user@test.com')
+        self.extra_data = {"expires": 3599, "auth_time": 1603287741, "token_type": "Bearer", "access_token": "ya29.EXAMPLE"}
```

Comment:
there's no reason for this to be a dictionary attached to the test class, just define these as constants at the top of the file

Reply:
Moved to a constant.